### PR TITLE
Renamed account and customer.

### PR DESCRIPTION
**Pull request by marcushanses**
- [x] https://github.com/payfunc/model-base/pull/19

### CODE REVIEW

File: Connection.ts [hunk 1]
```diff
@@ -74,7 +74,7 @@ export abstract class Connection {
 		Connection.keyValue = key
 		Connection.keyChanged.forEach(callback => callback(Connection.keyValue))
 	}
-	static get accountKeys(): authly.Token[] {
+	static get cusotmerKeys(): authly.Token[] {
```
Comment (florianebmeier) [inline Connection.ts]: customerKeys

File: ErrorMessage/translation/sv.ts [hunk 2]
```diff
@@ -29,7 +29,7 @@ export const sv = {
 		"Ytterligare autentisering krävs. Inställningarna för anslutningen till kortutfärdaren kan vara felaktiga.",
 	"An unknown error occured. Ensure that your card details are correct and try again.":
 		"Ett okänt fel uppstod. Kontrollera kortuppgifterna och försök igen.",
-	"The account could not be created.": "Kunde inte skapa konto.",
+	"The customer could not be created.": "Kunde inte skapa konto.",
```
Comment (florianebmeier) [inline ErrorMessage/translation/sv.ts:32]: Kunde inte skapa Kund

File: Order/index.ts [hunk 3]
```diff
@@ -39,11 +38,10 @@ export namespace Order {
 			(typeof value.number == "string" || value.number == undefined) &&
 			(typeof value.client == "string" || value.client == undefined) &&
 			isoly.DateTime.is(value.created) &&
-			(value.customer == undefined || Customer.is(value.customer)) &&
+			(value.customer == undefined || Contact.is(value.customer) || authly.Identifier.is(value.customerID, 16)) &&
```
Comment (florianebmeier) [inline Order/index.ts:41]: Not customerID

File: Order/index.ts [hunk 4]
```diff
@@ -72,7 +70,7 @@ export namespace Order {
 								value.client == undefined || { property: "client", type: "string | undefined" },
 							isoly.DateTime.is(value.created) || { property: "created", type: "DateTime" },
 							value.customer == undefined ||
-								Customer.is(value.customer) || { property: "customer", type: "Customer | undefined" },
+								Contact.is(value.customer) || { property: "customer", type: "Customer | undefined" },
```
Comment (florianebmeier) [inline Order/index.ts:73]: Flaw is missing case for customer:authly.Identifier

File: Order/index.ts [hunk 5]
```diff
@@ -315,10 +315,10 @@ export namespace Order {
 				new selectively.Type.String("invoice"),
 				new selectively.Type.String("defer"),
 				new selectively.Type.String("swish"),
-				new selectively.Type.String("account"),
+				new selectively.Type.String("customerID"),
```
Comment (florianebmeier) [inline Order/index.ts:318]: Remove

File: Order/index.ts [hunk 6]
```diff
@@ -315,10 +315,10 @@ export namespace Order {
...
 			]),
 		}),
-		account: new selectively.Type.String(),
+		customerID: new selectively.Type.String(),
```
Comment (florianebmeier) [inline Order/index.ts:321]: Remove

**Review by florianebmeier — CHANGES_REQUESTED** [review]

**Review by florianebmeier — APPROVED** [review]

File: Customer/Creatable.ts [hunk 7]
```diff
@@ -1,14 +1,14 @@
...
+import { Contact } from "@payfunc/model-base"
 import { Frequency } from "../Frequency"
 import { Schedule } from "../Schedule"
 import { Method } from "./Method"
 
 export interface Creatable {
 	id?: authly.Identifier
 	number?: string
-	customer?: Customer
+	customer?: Contact
```
Comment (nilssonemma) [inline Customer/Creatable.ts:11]: Rename this to contact?

**Review by nilssonemma — CHANGES_REQUESTED** [review]

File: Order/Creatable.ts [hunk 8]
```diff
@@ -56,7 +54,7 @@ export namespace Creatable {
 								value.number == undefined || { property: "number", type: "string | undefined" },
 							typeof value.client == "string" ||
 								value.client == undefined || { property: "client", type: "string | undefined" },
-							value.customer == undefined || Customer.is(value.customer) || Customer.flaw(value.customer),
+							value.customer == undefined || Contact.is(value.customer) || Contact.flaw(value.customer),
```
Comment (Magnus93) [inline Order/Creatable.ts:57]: Flaw is missing for customer is of type authly.Identifier

File: Customer/Creatable.ts [hunk 9]
```diff
@@ -1,14 +1,14 @@
...
 import { Frequency } from "../Frequency"
 import { Schedule } from "../Schedule"
 import { Method } from "./Method"
 
 export interface Creatable {
 	id?: authly.Identifier
 	number?: string
-	customer?: Customer
+	contact?: Contact
 	method: Method.Creatable[]
```
Comment (Magnus93) [inline Customer/Creatable.ts:13]: Remove empty row

**Review by florianebmeier — APPROVED** [review]

**Review by fredrikbryntesson — APPROVED** [review]

**Pull request merged**